.scss-lint.yml

```diff
@@ -1,3 +1,6 @@
+# This is an SCSS linter for the backend, using scss-lint
+# On the frontend, we are using stylelint-scss, whose
```

Comment:
Maybe a quick note on why we're using two different linters? Saw the slack thread, but might be useful to have in code.

Reply:
Good call!